### ISSUE TITLE
Running `make test` fails with Xcode 10.2

### ISSUE THREAD

**Issue by dfed** [airbnb/BuckSample#76]
I updated my machine to Xcode 10.2 today, and now I can't run `make test` locally. Here's the error I'm getting:

```
$ make test
tools/buck test //App:ExampleAppCITests --test-runner-env FOO=BAR
Building: finished in 15.6 sec (100%) 159/159 jobs, 49 updated
  Total time: 15.7 sec
Testing: finished in 6.7 sec
RESULTS FOR //App:ExampleAppCITests //App:UnitTests //App:UnitTestsWithHostApp //App:XCUITests
FAIL    <100ms  0 Passed   0 Skipped   1 Failed   Internal error from test runner
FAILURE Internal error from test runner main: Failed to query the list of test cases in the test bundle: 
stdout:

stderr:
This copy of libswiftCore.dylib requires an OS version prior to 12.2.0.
Child process terminated with signal 6: Abort trap

FAIL    <100ms  0 Passed   0 Skipped   1 Failed   Internal error from test runner
FAILURE Internal error from test runner main: Failed to query the list of test cases in the test bundle: 
stdout:

stderr:
This copy of libswiftCore.dylib requires an OS version prior to 12.2.0.
Child process terminated with signal 6: Abort trap

FAIL    <100ms  0 Passed   0 Skipped   1 Failed   Internal error from test runner
FAILURE Internal error from test runner main: Failed to query the list of test cases in the test bundle: 
stdout:

stderr:
This copy of libswiftCore.dylib requires an OS version prior to 12.2.0.
Child process terminated with signal 6: Abort trap

====TEST STATUS MESSAGES====
[2019-04-08 16:32:17.473][INFO] Collecting info for testables...
[2019-04-08 16:32:17.473][INFO] Collecting info for testables...
[2019-04-08 16:32:18.656][INFO] Collecting info for testables...
[2019-04-08 16:32:18.706][INFO] Collecting info for testables...
[2019-04-08 16:32:21.348][INFO] Collecting info for testables...
[2019-04-08 16:32:22.926][INFO] Collecting info for testables...
TESTS FAILED: 3 FAILURES
Failed target: //App:ExampleAppCITests
FAIL Internal error from test runner
Failed target: //App:UnitTests
FAIL Internal error from test runner
Failed target: //App:UnitTestsWithHostApp
FAIL Internal error from test runner
make: *** [test] Error 32
```

**Comment by thedavidharris** [comment]
Xcode 10.2 doesn't have the `/usr/lib/swift_static` directory anymore as part of it's toolchain I think so it won't statically link the Swift stdlib? Haven't dug through https://github.com/facebook/buck/blob/1e3437b3892ce9bb139ade9dc74fcf5aabc2709c/src/com/facebook/buck/swift/toolchain/impl/SwiftPlatformFactory.java#L47 much but I believe that's the cause.

Could try just adding `/usr/lib/swift` to the runpaths but there should be an out-of-the-box way for this to work.

**Comment by thedavidharris** [comment]
And there it is: https://github.com/facebook/buck/commit/1e3437b3892ce9bb139ade9dc74fcf5aabc2709c

**Comment by qyang-nj** [comment]
Quick solution is to add `-rpath /usr/lib/swift` to `cxx.ldflags` in `.buckconfig`.
Due to the commit @thedavidharris pointed above, I guess it's better to merge `facebook/buck` master into `airbnb/buck`.

**Comment by dfed** [comment]
@qyang-nj we last pulled from Facebook’s `master` in January. Probably worth updating. Our main branches on our fork right now are `buck-tiger-team` and `tiger-team-buck-sample`... probably worth renaming those to be more elucidating names while we’re at it.

**Comment by qyang-nj** [comment]
Fixed by #108
